DTS-STN/Alpha-Site
# Feature/filter experiments

# Description

[Filter for the experiments](https://trello.com/c/JEvAH1TO/168-filter-for-the-experiments)

**Description:** Give users an option to filter through experiment cards once there are many of them. 

**Done when:** We have the labels for the filters determined, can add states as we go

**Mock-ups**
![image](https://user-images.githubusercontent.com/77116011/117494479-61e48280-af42-11eb-871f-e85ea57dd085.png)
![image](https://user-images.githubusercontent.com/77116011/117494773-c869a080-af42-11eb-9319-7bb5531f9892.png)


**On the Branch**
![image](https://user-images.githubusercontent.com/77116011/117494652-9f491000-af42-11eb-8fc3-a7bfe5f49894.png)
![image](https://user-images.githubusercontent.com/77116011/117494695-b25be000-af42-11eb-99be-e6f155be8e02.png)



## Acceptance Criteria

We can add options and re-style as we go. The functionality will remain.

## Test Instructions

1. navigate to branch name: Feature/FilterExperiments
2. git pull and npm run dev
3. navigate to experiments page and see the filter options.
4. Check different screen widths.

## Help Requested

- I need help with testing. I have no idea how to test this. I only added the a11y unit test.
- The same applies for storybook. I am not sure how to set it up properly.

## Checklist


## Product and Sprint Backlog

[Filter for the experiments](https://trello.com/c/JEvAH1TO/168-filter-for-the-experiments)
